testing_app/integration_test/driver.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:integration_test/integration_test_driver.dart';
+
+Future<void> main() {
+  return integrationDriver(
+    responseDataCallback: (data) async {
+      // If the tests reported any data, save it to the disk.
+      if (data != null) {
+        for (var entry in data.entries) {
+          print('Writing ${entry.key} to the disk.');
+          // Default storage destination is the 'build' directory.
+          await writeResponseData(
```

Comment:
The docs say that the default `responseDataCallback` is `writeResponseData` - is there a reason you are overriding it to write separate files?

Reply:
For the performance tests, I have recorded the timeline as well as its summary. If I don't manually separate the data into different files, it gets stored into one single file which becomes huge. My current approach also allows the timeline to be opened using Chrome's tracing tools to further evaluate the performance.

Reply:
You should be able to load a trace into chrome without any additional code here. @dnfield does this sound correct?

Reply:
The raw trace should be loadable in `chrome://tracing`.

Reply:
For some reason, it isn't working for me.
Here's what I did to my current [integration-test](https://github.com/abd99/samples/tree/integration-test) branch:

1. Removed everything passed to `integrationDriver()` and kept everything else as it is.
2. Tried running the perf_test and then loading the `integration_response_data.json` file into `chrome://tracing`. 

The file was loaded, but no charts or data appeared.

I also tried removing `watchPerformance()` to make the reported data contain only timelines and not timeline summaries.

Reply:
I think the tracing tools aren't able to load multiple timelines at once. Am I missing something here?

Reply:
@abd99 Can you file an issue in the Flutter issue tracker?

Reply:
Sure. I'll create one shortly.

Reply:
Thanks